modules/data.atmosphere/R/upscale_met.R

```diff
@@ -0,0 +1,107 @@
+##' upscale_met upscales the temporal resolution of a dataset
+##' @name upscale_met
+##' @title upscale_met
+##' @export
+##' @param outfolder
+##' @param input_met - met dataset
+##' @param site.id
+##' @param reso - numeric value in HOURS that is the desired resolution you wish the dataset to be upscaled to
```

Comment:
At some point (doesn't have to be this PR) you might want to generalize this as it doesn't allow upscaling to monthly or annual (since those are not fixed length)